lib/puppet/provider/filesystem/aix.rb

```diff
@@ -161,7 +160,7 @@ def attribute_flag(pvalue)
       size: 'size',
       extended_attributes: 'ea',
       mount_options: 'options',
-      encrypted: 'efs',
+      encrypted: 'efs'
```

Comment:
I'm not a fan of this.  I think the vox pupuli rubocop config enforces a comma after the last element. If you now add a new element on the next line, you need to modify the previous line as well. That slightly increases the diff and can make rebases and reverts really tricky/ugly.

Reply:
Thanks @bastelfreak and totally understood. The CAT Teams practice is to fix all non-breaking rubocop violations, and this is a couple we regularly enforce across our modules.